tests/qiskit_convert_test.py

```diff
@@ -865,6 +867,32 @@ def test_controlled_unitary_conversion() -> None:
     assert np.allclose(u_qc, u_tkc)
 
 
+def test_qcontrol_box_conversion_to_qiskit() -> None:
+    ccch_001 = QControlBox(
+        Op.create(OpType.H), n_controls=3, control_state=(False, False, True)
+    )
+    cccs_110 = QControlBox(
+        Op.create(OpType.S), n_controls=3, control_state=(True, True, False)
+    )
+    cccRy_100 = QControlBox(
+        Op.create(OpType.Ry, 0.73), n_controls=3, control_state=(True, False, False)
+    )
+    ccU3_10 = QControlBox(
+        Op.create(OpType.U3, [0.1, 0.2, 0.3]), n_controls=2, control_state=(True, False)
+    )
+    circ1 = Circuit(4, name="test_circ")
+    circ1.add_gate(ccch_001, [0, 1, 2, 3])
+    circ1.add_gate(cccs_110, [0, 1, 2, 3])
+    circ1.add_gate(cccRy_100, [3, 2, 1, 0])
+    circ1.add_gate(ccU3_10, [1, 0, 2])
+    qc = tk_to_qiskit(circ1)
+    circ2 = qiskit_to_tk(qc)
+    DecomposeBoxes().apply(circ1)
+    DecomposeBoxes().apply(circ2)
+    assert circ1 == circ2
+    assert compare_unitaries(circ1.get_unitary(), circ2.get_unitary())
```

Comment:
I think this check of unitary equivalence may be redundant given the previous line

Reply:
Yes, can remove this. However, I think we should add a test that compares the unitary of the tket circuit with the unitary of the qiskit circuit (adjusted for endianness).

Reply:
added in [05dd936](https://github.com/CQCL/pytket-qiskit/pull/391/commits/05dd9361fa879c7cf713094310423e5081c63077)